graphs/README.md

```diff
@@ -16,7 +16,9 @@ An undirected graph only contains undirected edges.
 
 **Mixed graph** - A mixed graph is a graph that can contain both directed and undirected edges.
 
-**Walk** – A walk is a sequence of edges which joins a sequence of vertices.
+**Walk** – A walk is a sequence of vertices, such that any two consecutive vertices are joined by an edge.
+Note that two vertices are joined by a directed edge if and only if
+the direction of the edge matches the sequence order of the vertices.
```

Comment:
⚠️  Depending on how you define the direction of an edge, this condition is necessary, but not sufficient! In other words, "vertices joined by directed edge" implies "edge direction matches the order of vertices", but the converse is only true if the only way for the edge direction to match the order of those vertices is for the vertices to be the endpoints of the edge. That's why when I suggested this phrasing I said
```suggestion
Note that two vertices are joined by a directed edge only if
the direction of the edge matches the sequence order of the vertices.
```
(notice the lack of the first "if")